test/integration/targets/filesystem/tasks/setup.yml

```diff
@@ -29,6 +29,12 @@
     with_items:
       - python-xml
       - btrfsprogs
+
+  - name: install ocfs2 (Ubuntu)
+    package:
+      name: ocfs2-tools
+      state: present
+    when: ansible_os_family == 'Ubuntu'
```

Comment:
`ansible_distribution` must be used instead of `ansible_os_family`. Note that not `ansible_os_family == 'Debian'` could be used too.